Sources/FluentUI_iOS/Components/Notification/FluentNotification.swift

```diff
@@ -67,6 +67,10 @@ import SwiftUI
     ///
     /// If this property is nil, then this notification will use the background color defined by its design tokens.
     var backgroundGradient: LinearGradientInfo? { get set }
+
+    /// Performs an animation emphasizing the notification.
+    /// The animation alternates between upward and downward movements with spring physics.
+    @objc func bump()
```

Comment:
minor: This protocol is marked as `@objc` so I don't think it's needed here as well